api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -87,17 +87,24 @@ class HardwareManager(NamedTuple):
 
 
 class ProtocolContext(CommandPublisher):
-    """The Context class is a container for the state of a protocol.
+    """A context for the state of a protocol.
 
-    It encapsulates many of the methods formerly found in the Robot class,
-    including labware, instrument, and module loading, as well as core
-    functions like pause and resume.
+    The ``ProtocolContext`` class provides the objects, attributes, and methods that
+    allow you to configure and control the protocol.
 
-    Unlike the old robot class, it is designed to be ephemeral. The lifetime
-    of a particular instance should be about the same as the lifetime of a
-    protocol. The only exception is the one stored in
-    ``.legacy_api.api.robot``, which is provided only for back
-    compatibility and should be used less and less as time goes by.
+    Methods generally fall into one of two categories.
+
+      - They can change the state of the ``ProtocolContext`` object, such as adding
+        pipettes, hardware modules, or labware to your protocol.
+      - They can control the flow of a running protocol, such as pausing, displaying
+        messages, or controlling built-in robot hardware like the ambient lighting.
+
+    Do not instantiate a ``ProtocolContext`` directly for protocols that you plan to run
+    via the Opentrons App or touchscreen. The ``run()`` function of your protocol does
+    that for you. See the :ref:`Tutorial <run-function>` for more information.
```

Comment:
As far as I know, there's *never* any legitimate reason (or way) for a user to directly instantiate a `ProtocolContext`, so we can simplify this a little.

```suggestion
    Do not instantiate a ``ProtocolContext`` directly.
    The ``run()`` function of your protocol does that for you.
    See the :ref:`Tutorial <run-function>` for more information.
```